tests/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM debian:8
```

Comment:
Should we use minideb here?
And also change:

`RUN apt-get -q update && apt-get -qy install jq make curl`
for 
`RUN install_packages jq make curl`

Reply:
+1 wildo

Reply:
added at 9db8871, tests passing ok && merging.